common/components/AddressField.tsx

```diff
@@ -35,7 +35,7 @@ const AddressField: React.SFC<Props> = ({
   showLabelMatch,
   toChecksumAddress,
   showIdenticon,
-  placeholder = donationAddressMap.ETH,
+  placeholder = 'donate.mycryptoid.eth or 0x4bbeEB066eD09B7AEd07bF39EEe0460DFa261520',
```

Comment:
We can continue using `donationAddressMap` to make this easier to update.
```suggestion
  placeholder = `donate.mycryptoid.eth or ${donationAddressMap.ETH}`,
```
It may also be helpful to move `'donate.mycryptoid.eth'` to the config if it's going to be used in other places.